### PR TITLE
support thoth-prescription job with the argocd

### PR DESCRIPTION
**Pull request by harshad16**
support thoth-prescription job with the argocd
Signed-off-by: Harshad Reddy Nalla <hnalla@redhat.com>

### CODE REVIEW

**Review by anishasthana — APPROVED** [review]
/lgtm

File: envs/moc/zero/thoth/prod/prod-thoth-prescription-sync.yaml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+metadata:
+  name: prod-thoth-prescription-sync
+spec:
+  project: thoth
+  source:
+    repoURL: 'https://github.com/thoth-station/thoth-application.git'
+    path: prescription-sync-job/overlays/cnv-prod
+    targetRevision: master
+  destination:
+    server: 'https://api.ocp4.prod.psi.redhat.com:6443'
```
Comment (tumido) [inline envs/moc/zero/thoth/prod/prod-thoth-prescription-sync.yaml:13]: @harshad16 ArgoCD in Operate First can't manage workloads within PSI. Please either use an Operate First cluster or deploy this to https://github.com/AICoE/aicoe-cd

**Review by tumido — CHANGES_REQUESTED** [review]

File: envs/moc/zero/thoth/prod/prod-thoth-prescription-sync.yaml (hunk 1 above)
Comment (harshad16) [reply envs/moc/zero/thoth/prod/prod-thoth-prescription-sync.yaml]: Thanks for pointing. i made the copy-paste mistake, i have fixed it.

**Review by tumido — COMMENTED** [review]
/lgtm

**Review by tumido — APPROVED** [review]
/approve

**Pull request merged**